tests/test_other.py

```diff
@@ -9872,3 +9884,19 @@ def test_split_main_module(self):
     self.assertNotIn('profile', result)
     self.assertIn('Hello from main!', result)
     self.assertIn('Hello from lib!', result)
+
+
+  def test_split_dwarf(self):
+    # TODO(sbc): test the dwo is correct/usable
+    self.run_process([EMCC, '-c', path_from_root('tests', 'core', 'test_hello_world.c'), '-g', '-gsplit-dwarf'])
+    self.assertExists('test_hello_world.o')
+    self.assertExists('test_hello_world.dwo')
+    obj_out = read_sections_using_objdump('test_hello_world.o')
+    dwo_out = read_sections_using_objdump('test_hello_world.dwo')
+    obj_info = [s for s in obj_out if s[1] == '.debug_info'][0]
+    dwo_info = [s for s in dwo_out if s[1] == '.debug_info.dwo'][0]
+    obj_size = int(obj_info[2], 16)
+    dwo_size = int(dwo_info[2], 16)
+    print('.debug_info: %d' % obj_size)
+    print('.debug_info.dwo: %d' % dwo_size)
+    self.assertGreater(dwo_size, obj_size)
```

Comment:
Actually I guess if we're going so far as to look at the sections we could run `llvm-dwarfdump'. But this seems ok for now, better than nothing.